ios/RCTWebRTC/WebRTCModule.m

```diff
@@ -54,6 +56,11 @@ - (instancetype)init
     _peerConnections = [NSMutableDictionary new];
     _localStreams = [NSMutableDictionary new];
     _localTracks = [NSMutableDictionary new];
+      
+    dispatch_queue_attr_t attributes =
+    dispatch_queue_attr_make_with_qos_class(DISPATCH_QUEUE_SERIAL,
+                                            QOS_CLASS_USER_INITIATED, -1);
+    _workerQueue = dispatch_queue_create("WebRTCModule.queue", attributes);
```

Comment:
Since we create this in `init`, we should destroy it in `dealloc`. IRL this init/dealloc cycle happens only once, but let's try to be as tidy as possible.

Reply:
With ARC, using dealloc and releasing ivars like this one is not longer required.
http://releases.llvm.org/5.0.0/tools/clang/docs/AutomaticReferenceCounting.html#dealloc